### PR TITLE
Stash Cache container doc review

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/data/stashcache/run-stashcache-container.md [hunk 1]
```diff
@@ -14,52 +15,55 @@ Before Starting
...
-      when usage goes above the high watermark, the cache deletes until it hits the low watermark
+- `XC_SPACE_HIGH_WM`, `XC_SPACE_LOW_WM`: High-water and low-water marks for disk usage;
+      when usage goes above the high-water mark, the cache will delete files until it hits the low-water mark.
 - `XC_PORT`: TCP port that XCache listens on
 - `XC_RAMSIZE`: Amount of memory to use for storing blocks before writting them to disk. (Use higher for slower disks).
-- `XC_BLOCKSIZE`: The size of the blocks in the cache
+- `XC_BLOCKSIZE`: Size of the blocks in the cache.
 - `XC_PREFETCH`: Number of blocks to prefetch from a file at once.
-       This is a value of how aggressive the cache is to request portions of a file. Set to `0` to disable
+       This controls how aggressive the cache is to request portions of a file. Set to `0` to disable.
```
Comment (efajardo) [inline docs/data/stashcache/run-stashcache-container.md:47]: I would mention the recommended value is not to disable.

File: docs/data/stashcache/run-stashcache-container.md [hunk 2]
```diff
@@ -14,52 +15,55 @@ Before Starting
...
 
-### Disabling OSG monitoring ###
+### Disabling OSG monitoring (testing only) ###
+
+!!! warning
+    Only disable OSG monitoring on services that are solely used for testing.
```
Comment (efajardo) [inline docs/data/stashcache/run-stashcache-container.md:52]: I think @brianhlin suggested to get rid of this for the origin so same thing for the cache.

File: docs/data/stashcache/run-stashcache-container.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/data/stashcache/run-stashcache-container.md]: Done

File: docs/data/stashcache/run-stashcache-container.md (hunk 2 above)
Comment (matyasselmeci) [reply docs/data/stashcache/run-stashcache-container.md]: Done

File: docs/data/stashcache/run-stashcache-container.md [hunk 3]
```diff
@@ -164,26 +157,26 @@ For caches that store over `10 TB` or that have assigned space for storing the c
...
+Validating the Cache
 ---------------------
 
-For example, if you've chosen `8212` as your host port, you can verify that it worked with the command:
+The cache server functions as a normal HTTP server and can interact with typical HTTP clients, such as `curl`.
+
 
 ```console
-user@host $ curl http://localhost:8212/user/dweitzel/public/blast/queries/query1
+user@host $ curl -O http://cache_host:8000/osgconnect/public/rynge/test.data
```
Comment (brianhlin) [inline docs/data/stashcache/run-stashcache-container.md:174]: Shouldn't the port be based off of `<HOST PORT>` from the above `docker run` command?

Comment (matyasselmeci) [reply docs/data/stashcache/run-stashcache-container.md:174]: Yeah that text kinda got lost. Let me fix...

**Pull request merged**